dist_app_linux.tcjs

```diff
@@ -1,21 +1,20 @@
 let tc = TCF.define(TCF.CPP_TRANSFORM);
-tc.pocoLoc = '/home/tests/poco';
+//tc.pocoLoc = '/home/tests/poco';
 tc.sources = [
     'platform:/resource/DistributedPingPong/CPPModel.emx#_qJc4wfMMEeiMwvLnGvgCsg'
 ];
 tc.createTargetProject = true;
 tc.targetProject = '/DistributedPingPong_target';
 tc.prerequisites = [
-    'platform:/resource/LibTcpServer/tcpServerLib.tcjs'
+    'platform:/resource/LibTcpServer/tcpServerLib_linux.tcjs'
 ];
 tc.compilationMakeType = MakeType.GNU_make;
-tc.compileArguments = '$(DEBUG_TAG) -fexceptions';
+tc.compileArguments = '$(DEBUG_TAG)';
 tc.inclusionPaths = [
```

Comment:
inclusionPaths are empty now, remove this property completely

Reply:
ok, I will remove it.